main.css

```diff
@@ -16,10 +16,12 @@
 }
 
 .home-section-pic {
-    width: 80%;
-    margin-left: 26rem;
+  max-width: 100%;
+  /* margin-left: 26rem; */
```

Comment:
Can remove commented out line. The more commented out code, the bigger the file gets unnecessarily. If we need to review what was here, we can use the commits :)

Reply:
I agree.